### PR TITLE
#18 - Introduced standard config.file location

### PR DESCRIPTION
**Pull request by xamanu**

### CODE REVIEW

File: osm2gtfs.py [hunk 1]
```diff
@@ -29,11 +30,13 @@
...
-    try:
-        config = json.load(args.config)
-    except ValueError, e:
-        print('Failed to load config json file:')
-        print(e)
+    if os.path.isfile('config.json'):
+        with open("config.json") as json_file:
+            config = load_config(json_file)
+    elif args.config is not None:
+            config = load_config(args.config)
```
Comment (grote) [inline osm2gtfs.py]: What happens here if the config file as been specified as an argument, but does not exist?

**Comment by grote** [comment]
Thanks for this PR! :)

Unfortunately, I can't merge it right away because it conflicts with the other one I just merged.

File: osm2gtfs.py (hunk 1 above)
Comment (xamanu) [inline osm2gtfs.py]: > usage: osm2gtfs [-h] [--config FILE] [--output FILENAME]
                [--refresh-route ROUTE | --refresh-all-routes | --refresh-all-stops | --refresh-all]
osm2gtfs: error: argument --config/-c: can't open 'foo.json': [Errno 2] No such file or directory: 'foo.json'

**Comment by xamanu** [comment]
I re-rolled the commit. Hopefully it works now. Thanks for feedback and pulling in.

**Pull request merged**